extra/callback_server.py

```diff
@@ -0,0 +1,30 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import sys
+
+
+class RequestHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        content_length = int(self.headers['Content-Length'])
+        post_data = self.rfile.read(content_length)
+
+        print("Received POST request:")
```

Comment:
Instead of print, maybe an app logger of some sort?

Reply:
since we only want to run this locally, I prefer to keep it as a light weight independent script for running local applications without the need to install/import any custom/additional libs. I don't think the print instead of logger will cause any issues?

Reply:
That's fine for now - if we add anything else I think adding it as a logger would be appropriate - it's a standard library so wouldn't add any more dependencies, FWIW